### ISSUE TITLE
[Enhancement] Calendar should have a max-width instead of following the viewport

### ISSUE THREAD

**Issue by MartinSkovsen** [kirbydesign/designsystem#1398]
**Please add a short description of your enhancement request**
I would like the width of the calendar to have a fixed max-width instead of following the viewport as it currently does.

**Describe the solution you'd like**
Give it a maximum allowable width of 656px.

**Describe alternatives you've considered**
As I understand it, we are currently considering whether to let the calendar follow the viewport or a max-width.

**Additional context**
Cookbook: https://cookbook.kirby.design/home/showcase/calendar
Zeplin: https://zpl.io/VKOWZ5X

### Kick Off:

- [ ] Ensure the enhancement is `UX refined` and aligned with UX 
_The component and/or enhancement should be published and available in the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)_
- [ ] Ensure the enhancement has been `Tech refined` with @kirbydesign/kirby-guild and this issue is updated with a clear implementation description  
_This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation)_
- [ ] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code:

- [ ] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
- [ ] Create a draft implementation and push to Github
- [ ] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request
- [ ] Implement unit tests
- [ ] Update Cookbook Examples and Showcase, i.e. see [Radio](https://cookbook.kirby.design/home/showcase/radio)  
_Also remember to add any relevant new API documentation_

### Review:

- UX review:
  - [ ] Ensure implementation is correct in relation to the UX design and the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)
  - [ ] With UX agree on the version of the implementation
- Code review:
  - [ ] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review  
_Remember to add `closes #issueno` to the description of the PR._
  - [ ] Once approved, merge feature branch/PR to master
- [ ] Ask a member of @kirbydesign/kirby-guild to add a link to component showcase from Kirby Component Status and update the version number

:tada: Celebrate

**Comment by MartinSkovsen** [comment]
@henrikvoetmand Can you provide some context and, if needed, edit the issue description? This needs UX refinement :)

**Comment by MartinSkovsen** [comment]
This issue needs further refinement between UX and tech (@henrikvoetmand & @bommariusser): 
- Should the calendar come wrapped in a card?
- Should the max-width be applied to the card or the calendar itself?
- Are there use cases where the calendar is used without a card? I.E.: In a drawer?

**Comment by RasmusKjeldgaard** [comment]
@henrikvoetmand and I discussed this, and concluded that the calendar should not have a fixed max width. 
It is rather the responsibility of the UX guidelines to explain that we recommend the wrapper around the calendar to have a max width on larger screens. 

I have created a new issue for referencing UX guidelines in the cookbook. : #1434 

So to conclude: 
- Should the calendar come wrapped in a card? 
    - No, we want it to be flexible and reusable throughout all wrapper-components. 
- Should the max-width be applied to the card or the calendar itself?
   - There should be no max width - Imagine if you were to put the calendar in e.g. a card with a width larger than the maximum allowed for the calendar. Then the calendar wouldn't adapt to the full width of the card, which would make for a poor developer experience. 
- Are there use cases where the calendar is used without a card? I.E.: In a drawer?
   - Yes, or at least we want to keep support for this in the future.
